operating/respond-to-violations.adoc

```diff
@@ -0,0 +1,42 @@
+[id="respond-to-violations"]
+= Responding to violations
+include::modules/common-attributes.adoc[]
+:context: respond-to-violations
+
+toc::[]
+
+[role="_abstract"]
+Using {product-title} you can view policy violations, drill down to the actual cause of the violation, and take corrective actions.
+
+{product-title} built-in policies identify a variety of security findings, including vulnerabilities (CVEs), violations of DevOps best practices, high-risk build and deployment practices, and suspicious runtime behaviors.
+Whether you use the default out-of-box security policies or use your own custom policies, {product-title} reports a violation when an enabled policy fails.
+
+include::modules/violations-view.adoc[leveloffset=+1]
+
+[id="view-violation-details"]
+== Viewing violation details
+When you select a violation in the *Violations* view, the *Violation Details* open in a panel on the right.
```

Comment:
the way this is worded sounds odd with *Violation Details* being styled as the name of the panel. I expected to see "page" or "panel" after it, with the "the" in front of it. Suggest rewording a little, maybe:

```
the *Violation Details* panel opens on the right.
```
or
```
the violation details open in a panel on the right.
```